### PR TITLE
Separate Helm setup and GKE credential setup

### LINKED ISSUE #14: Separate GKE authentication from tools setup in Helm actions
Tools:
- kfilt
- kubectl
- Helm
- ...

### PR DESCRIPTION
**Pull request by yannick-roeder**
Fixes #14

### CODE REVIEW

File: .github/workflows/helm-gke-destroy.yaml [hunk 1]
```diff
@@ -55,20 +55,24 @@ jobs:
       - name: Checkout Repository
         uses: actions/checkout@v3
 
-      - name: Setup Environment
-        uses: bakdata/ci-templates/actions/helm-gke-setup@v1.6.0
+      - name: Setup Credentials
+        uses: bakdata/ci-templates/actions/setup-credentials@feat/separate-helm-credentials-setup
```
Comment (yannick-roeder) [inline .github/workflows/helm-gke-destroy.yaml:59]: ToDo: pre-bump version everywhere

File: actions/helm-setup/action.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  helm-version:
+    description: "Helm version"
+    default: "v3.10.1"
+    required: false
+runs:
+  using: "composite"
+  steps:
+    # TODO: replace by ci runner image
+    - name: Setup python
+      uses: actions/setup-python@v1
```
Comment (disrupted) [inline actions/helm-setup/action.yaml:21]: what does this action need Python for?

Comment (yannick-roeder) [reply actions/helm-setup/action.yaml:21]: I think GCloud needs it, I will check again to make sure

Comment (yannick-roeder) [reply actions/helm-setup/action.yaml]: It is not needed, I removed it

**Review by disrupted — APPROVED** [review]

File: .github/workflows/kustomize-gke-deploy.yaml [hunk 3]
```diff
@@ -51,18 +46,21 @@ jobs:
   deploy:
     runs-on: ubuntu-22.04
     steps:
-      - name: Checkout repository
-        uses: actions/checkout@v2
+      - name: Checkout Repository
```
[the remark this answers is not in the document]
Comment (yannick-roeder) [reply .github/workflows/kustomize-gke-deploy.yaml:49]: Didn't you mean that I should capitalize all step names with your previous comment?

[the remark this answers is not in the document]
Comment (disrupted) [reply .github/workflows/kustomize-gke-deploy.yaml:49]: I meant that names such as _Helm_ should be capitalized

[the remark this answers is not in the document]
Comment (yannick-roeder) [reply .github/workflows/kustomize-gke-deploy.yaml:49]: Ok, then I will correct it again

**Pull request merged**